unpackaged/config/perf/objects/___NAMESPACE___DataImport__c.object

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>Qualified_Date__c</fullName>
+        <description>The date your organization identified this donor as qualifying for this Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Qualified_Date__c</inlineHelpText>
+        <label>Qualified Date</label>
+        <required>false</required>
+        <trackFeedHistory>false</trackFeedHistory>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>npe01__Do_Not_Automatically_Create_Payment__c</fullName>
+        <description>????</description>
+        <externalId>false</externalId>
+        <inlineHelpText>npe01__Do_Not_Automatically_Create_Payment__c</inlineHelpText>
+        <label>Do Not Automatically Create Payment</label>
+        <required>false</required>
+        <trackFeedHistory>false</trackFeedHistory>
+        <trackTrending>false</trackTrending>
+        <type>????</type>
```

Comment:
Something like this:

        <type>Picklist</type>
        <valueSet>
            <valueSetDefinition>
                <sorted>false</sorted>
                <value>
                    <fullName>True</fullName>
                    <default>false</default>
                    <label>True</label>
                </value>
                <value>
                    <fullName>False</fullName>
                    <default>false</default>
                    <label>False</label>
                </value>
            </valueSetDefinition>
        </valueSet>